assets/src/blocks/Spreadsheet/Spreadsheet.js

```diff
@@ -16,22 +16,38 @@ export class Spreadsheet extends Component {
   renderEdit() {
     const { __ } = wp.i18n;
 
-    const { color } = this.props;
+    const { attributes, setAttributes } = this.props;
+
+    const toAttribute = attributeName => value => {
```

Comment:
(Non-blocking comment): This one is already defined in Covers and Articles, maybe we can move it somewhere like in some kind of "block helpers" folder/file?

Reply:
We can, though the tricky bit is that it currently depends on `setAttributes` from the scope where it's defined. So if we isolate it then it would need to get that as an argument, like `setAttributes => attributeName => value => { setAttributes( { [ attributeName ]: value } ) }`. But I didn't do that yet because maybe we can look into a way to define this as a HOC or something else together with reading the attribute value, so we can avoid the following (having to provide the attribute name everywhere in 2 props, once as a string, once as a property):
```
            value={ attributes.url }
            onChange={ toAttribute( 'url' ) }
```